packages/expo-cli/src/commands/info/doctor/index.ts

```diff
@@ -5,7 +5,7 @@ import { applyAsyncActionProjectDir } from '../../utils/applyAsyncAction';
 export default function (program: Command) {
   applyAsyncActionProjectDir(
     program
-      .command('doctor')
+      .command('doctor [path]')
```

Comment:
This is a breaking change as you can no longer run `expod doctor ./my-app --fix-dependencies`

Reply:
I couldn't get that to work anyway, so I assumed it didn't. But it's restored now (and still not working for me)